src/lib/axes.typ

```diff
@@ -30,6 +30,18 @@
     stroke: (paint: gray, dash: "dotted"),
     fill: none
   ),
+  x: (
```

Comment:
If we add `x` and `y`, we should also add `x2` and `y2`.

Reply:
There is no x2 and y2 but top, right, bottom and left. Added the defaults within the first axis-settings loop.